product_packaging_unit_price_calculator/wizards/product_package_price.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_product_package_price_form" model="ir.ui.view">
+        <field name="name">Product Package Price</field>
+        <field name="model">product.package.price.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Product Packaging Price">
+                <field name="product_tmpl_id" invisible="1" />
+                <field name="product_variant_ids" invisible="1" />
```

Comment:
[18] Fields that are required by Python expressions (invisible, column_invisible, readonly, required, context, domain) are now added automatically as invisible and readonly

recommend removed